http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/SamDAO.scala

```diff
@@ -71,6 +73,8 @@ trait SamDAO[F[_]] {
   ): F[Option[UserSubjectId]]
 
   def getUserSubjectIdFromToken(token: String)(implicit ev: Ask[F, TraceId]): F[Option[UserSubjectId]]
+
+  def getLeoAuthToken: F[Authorization]
```

Comment:
hmm... it seems odd that this is an exposed trait of the SamDAO. I don't see it being used external to the HttpSamDao?

Reply:
it's used https://github.com/DataBiosphere/leonardo/pull/2581/files#diff-8af5da8d5edb93397a2e5c102c53047173dac803b46343c12568196a90cee553R211...I comment out that part for now becuz we in fiabs, sam is not started yet, so this will fail Leonardo's bootup.